lib/src/charts/data/bars/simple_bar.dart

```diff
@@ -17,7 +21,33 @@ class SimpleBar extends BarGroup with EquatableMixin {
     super.groupStyle,
   });
 
+  factory SimpleBar.zero() {
+    return SimpleBar(
+      xValue: 0,
+      yValue: BarData.zero(),
+    );
+  }
+
   @override
   List<Object?> get props =>
       [super.xValue, yValue, super.label, super.labelStyle, super.groupStyle];
+
+  static SimpleBar lerp(BarGroup? current, BarGroup target, double t) {
+    if ((current is SimpleBar?) && target is SimpleBar) {
+      return SimpleBar(
+        xValue: lerpDouble(current?.xValue, target.xValue, t) as num,
+        yValue: BarData.lerp(current?.yValue, target.yValue, t),
+        label: target.label,
+        labelStyle:
+            ChartTextStyle.lerp(current?.labelStyle, target.labelStyle, t),
+        groupStyle:
+            BarDataStyle.lerp(current?.groupStyle, target.groupStyle, t),
+      );
+    } else {
+      throw Exception('Both current & target data should be of same series!');
```

Comment:
In the else case we should lerp from null to target. Can be skipped for now but will be needed for multi chart